source/about/devops-collaboration.rst

```diff
@@ -0,0 +1,22 @@
+:nosearch:
+
+.. This page is intentionally not accessible via the LHS navigation pane because it's common content included on other docs pages.
+
+DevOps collaboration
+--------------------
+
+In DevOps, teams work together to accelerate software development and deployment processes, reduce costs, and ensure their organization remains competitive in the fast-paced digital marketplace. DevOp practitioners include a variety of disciplines including product managers, designers, developers, operations, quality assurance, security, and project managers. 
+
+.. image:: ../images/devops-agile.png
+  :alt: This content is suitable for these roles.
+
+DevOps teams can experience the following collaboration pains: 
+
+- Members from different departments and timezones have different priorities, skills, and vocabularies, which can lead to misunderstandings, miscommunications, and delays in decision-making.
+- Limited visibility into each other's work can lead to duplication of effort, conflicts, and missed opportunities for optimization.
+- Resistance from stakeholders who are reluctant to adopt new technologies or processes can slow down innovation, increase costs, and create tension within the team.
+- Difficulty ensuring that software is developed, deployed, and managed securely to protect against data breaches and cyber-attacks. This involves coordinating efforts between developers, operations, and security teams to identify and mitigate security risks.
+
+There are several different processes that DevOps teams need to effectively collaborate in order to deliver software reliably. Each requires a different set of procedures and tools, but they all share the common goal of shipping high-quality software to meet customer needs and expectations. Mattermost can help solve collaboration pains in many DevOps use cases. 
+
+- Learn how `Agile Software Development <https://academy.mattermost.com/courses/enrolled/2075064>`_ can help Technical teams work together to develop software or applications. Collaboration can include sharing code, working together on a common platform, and identifying and resolving technical issues.
```

Comment:
I suspect we need to update this URL to point to the course itself, rather than the enrolled page. We'll likely need to publish the new content on Academy first.